client/app/dedicated/server/translations/Messages_fr_FR.xml

```diff
@@ -479,7 +479,7 @@
    <translation id="server_configuration_ftpbackup_add_allow" qtlid="141795">Autoriser</translation>
    <translation id="server_configuration_ftpbackup_add_deny" qtlid="141808">Refuser</translation>
    <translation id="server_configuration_ftpbackup_add_ip" qtlid="180850">Block concerné :</translation>
-   <translation id="server_configuration_ftpbackup_add_ip_length_error">Limite de blocs sélectionnés atteinte</translation>
+   <translation id="server_configuration_ftpbackup_add_ip_length_error">Limite de blocs sélectionnés atteinte (50 maximum)</translation>
```

Comment:
This number could change in the future, maybe you could have a named parameter instead

Reply:
It is in discussion to maybe remove it, because it has no sense to say this number

Reply:
Ok so the translation will include a number, so yes, using a param is better :)